Model/Sync/Orders/Main.php

```diff
@@ -163,4 +163,34 @@ public function insertOrUpdateYotpoTableData($data)
         ];
         $this->insertOnDuplicate('yotpo_orders_sync', $finalData);
     }
+
+    /**
+     * @param DataObject $response
+     * @return array <mixed>
+     */
+    public function getMissingProductIds($response)
```

Comment:
Should be something like `getMissingProductIdsFromNotFoundResponse`.

Reply:
This is done